Source/QuantityTypes.Tests/QuantityTypes.Tests.csproj

```diff
@@ -89,7 +90,9 @@
       <Name>QuantityTypes</Name>
     </ProjectReference>
   </ItemGroup>
-  <ItemGroup />
+  <ItemGroup>
+    <Service Include="{82A7F48D-3B50-4B1E-B82E-3ADA8210C358}" />

```

Comment:
I guess it is a test project tag as described in
http://stackoverflow.com/questions/18614342/what-is-service-include-in-a-csproj-file-for
https://connect.microsoft.com/VisualStudio/feedback/details/800245/vs2013rc-adds-to-vs2012-c-project-section-itemgroup